resources/scripts/components/index.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Calls a supplied function once
+ * document has loaded.
+ *
+ * @param  {CallableFunction} onReady
+ * @return {void}
+ */
+export const domReady = (onReady) => {
+  window.requestAnimationFrame(function check() {
+    document.body ? onReady() : window.requestAnimationFrame(check);
+  });
+};
```

Comment:
```suggestion
export const domReady = (onReady) => {
  window.requestAnimationFrame(document.body ? onReady : domReady);
}
```

We can check for the body immediately, right? Do we even need to do that in an animation frame the first time it’s called? I don’t know enough about `window.requestAnimationFrame`, so I could be wrong.

As an aside, I think we should change the name of this function to `bodyIsReady` (alternatively prefixed with `my`), since it kind of breaks the mold of DOM readiness functions (particularly by going outside of the definition of `DOMContentLoaded`).